packages/editor/src/components/save-publish-panels/style.scss

```diff
@@ -11,7 +11,7 @@
 	box-sizing: border-box;
 	width: $sidebar-width;
 	background-color: $white;
-	border: 1px dotted $gray-300;
+	border: $border-width dotted $gray-300;
 	height: auto !important; // Need to override the default sidebar positionnings
```

Comment:
I'm sorry for widening the rabbit hole! The CSS comment here both has a typo, and could use a period at the end. Something like `// Necessary to override the default sidebar positioning.`